### ISSUE TITLE
The prop value with an expression type of JSXEmptyExpression could not be resolved

### ISSUE THREAD

**Issue by Ali-Waseer** [jsx-eslint/jsx-ast-utils#128]
I am facing an issue while I building my react project.
The prop value with an expression type of JSXEmptyExpression could not be resolved. Please file an issue ( https://github.com/jsx-eslint/jsx-ast-utils/issues/new ) to get this fixed immediately

**Comment by ljharb** [comment]
Do you know what code is causing this error?

**Comment by Ali-Waseer** [comment]
No, actually I just want to build my project and want it to live. The other
developer working on same project and they are building and deploying their
changes. They don't have such issues like this.

**Comment by ljharb** [comment]
If you’re working on the same code with the same dep graph, you should get the same results.

i can’t help you fix your problem without more information.

**Comment by Ali-Waseer** [comment]
So, what you need more information?

**Comment by ljharb** [comment]
The code that’s causing the linting error, and confirmation that you’re using the latest version/

**Comment by happymvp** [comment]
@ljharb I got the issue from this empty onKeyDown handler in my case
`  <div onClick={handleCount} onKeyDown={} role="button" tabIndex={0}>
      {count}
    </div>`

**Comment by ljharb** [comment]
@happymvp thanks - that's invalid jsx tho, i believe, so it shouldn't even parse. what eslint parser are you using?

**Comment by happymvp** [comment]
> [@happymvp](https://github.com/happymvp) thanks - that's invalid jsx tho, i believe, so it shouldn't even parse. what eslint parser are you using?

@typescript-eslint/parser

**Comment by ljharb** [comment]
hm, ok thanks, i'll see what i can do. i think there's a few places TS allows invalid JSX and this may be one of them.
